script.js

```diff
@@ -1,11 +1,22 @@
+
 // Create an array called `colors` that contains three colors.
+var colors = ['red', 'green', 'blue'];
 
 // Log the last item in the array to the console.
-
+colors[2];
+console.log("blue");
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+var colors();
+for ( i in colors) {
+  console.log("My favorite color is", colors[2]);
```

Comment:
This will only print out the final color in the array. You need to make use of `i`.